paddle/fluid/operators/math/blas.h

```diff
@@ -90,6 +101,28 @@ class Blas {
                    int K, T alpha, const T* A, const T* B, T beta, T* C,
                    int batchCount, int64_t strideA, int64_t strideB) const;
 
+  template <typename T>
```

Comment:
Other methods in class template Blas are all in blas_impl.h, why not more the body of this new MatMul into blas_impl.h as well?

Reply:
Done.